### PR TITLE
add BitToggleButton documentation (#719)

### LINKED ISSUE #719: Add BitToggleButton component Documentation Comments
This issue is based on issue #437

Add a summary for each parameter in the BitToggleButton component.

### PR DESCRIPTION
**Pull request by Md23Mh**
This closes #719

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs [hunk 1]
```diff
@@ -11,12 +11,23 @@ public partial class BitToggleButton
...
         [Parameter] public string? AriaDescription { get; set; }
+
+        /// <summary>
+        /// If true, add an aria-hidden attribute instructing screen readers to ignore the element.
+        /// </summary>
         [Parameter] public bool AriaHidden { get; set; }
 
         /// <summary>
-        /// determine if the button is checked state, default is true
+        /// Determine if the button is checked state, default is true.
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs:30]: "... is in checked ..."

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs [hunk 2]
```diff
@@ -31,35 +42,49 @@ public bool IsChecked
             }
         }
 
+        /// <summary>
+        /// Callback that is called when the IsChecked parameter changed.
+        /// </summary>
         [Parameter] public EventCallback<bool> IsCheckedChanged { get; set; }
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs:48]: this is not a correct comment for this parameter.
for now remove the documentation comment here

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs [hunk 3]
```diff
@@ -31,35 +42,49 @@ public bool IsChecked
...
         [Parameter] public string? Target { get; set; }
 
         /// <summary>
-        /// The title to show when the mouse is placed on the button
+        /// The title to show when the mouse is placed on the button.
         /// </summary>
         [Parameter] public string? Title { get; set; }
 
+        /// <summary>
+        /// Toggles the state when the button is clicked.
```
Comment (msynk) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs:76]: please change this line to something like "A callback that is called when the button is clicked"

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs (hunk 2 above)
Comment (Md23Mh) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs:48]: Should I create a new issue for this one? so we won't forget it. Or would you give me a hint for a better description?

Comment (msynk) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs:45]: this event is for the 2-way bound parameter which is related to the "@bind-" notation in blazor

Comment (msynk) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs:45]: as I said, for now, there is no need for the documentation comment here.

**Review by msynk — APPROVED** [review]

**Pull request merged**